dev-util/capstone/capstone-4.0.1.recipe

```diff
@@ -45,10 +45,16 @@ BUILD_REQUIRES="
 	"
 BUILD_PREREQUIRES="
 	cmd:awk
+	cmd:find
 	cmd:gcc$secondaryArchSuffix
 	cmd:make
+	cmd:unzip
+	cmd:wget
 	"
 
+fuzz_test_url="https://github.com/groundx/capstonefuzz"
```

Comment:
This can be removed, next line should be moved into TEST()

Reply:
Done.